test/testUtils.js

```diff
@@ -0,0 +1,40 @@
+// Core
+import React, { useContext } from 'react';
+// Utils
+import i18njs from '@k14v/i18njs';
+// Context
+import I18nContext from '../src/I18nContext';
+
+
+export const I18N_DATA = {
+  locale: 'en',
+  locales: {
+    es: {
+      'esto es una prueba': 'Esto es una prueba',
+      Teléfono: 'Teléfono',
+      Perro: 'Perro',
+    },
+    en: {
+      'esto es una prueba': 'This is a test',
+      Teléfono: 'Phone',
+      Perro: 'Dog',
+    },
+    de: {
+      'esto es una prueba': 'Das ist ein Test',
+      Teléfono: 'Telefon',
+      Perro: 'Hund',
+    },
+  },
+};
+
+export function fetchI18n() {
+  return new Promise((resolve) => {
+    const i18n = i18njs(I18N_DATA);
```

Comment:
I will leave a issue to change this in a future by the newer release of `i18njs` [0.2.0](https://github.com/k14v/i18njs/pull/21)